src/windows/export.py

```diff
@@ -179,6 +179,34 @@ def __init__(self):
             self.channel_layout_choices.append(layout[0])
             self.cboChannelLayout.addItem(layout[1], layout[0])
 
+        # Add streamable file format presets (mp4/mov only)
+        #
+        # 0 - none
+        # 1 - faststart
+        # 2 - fragmented
+        #
+        streamable_presets = [_("Disable"), _("Place index table to the beginning"), _("Make selfcontained fragmented file")]
```

Comment:
About short names for selections - I will think about it.

About HDMI-CEC - Sony has incompatible interface with Dune HD for example (or has limited set of supported functions), so it is good that it named differently - because it doesn't works as intended (you need to buy gadgets of one brand). The good news is that you'd make you brain to work a bit when reading these options names...

About `selfcontained` - if `self-contained` is right word I prefer to use it. Because the mentioned earlier ISO standard allows to split media into files (or even into streams) that stored separately but uses one header file (kind of "master"). Or program can split export file to workaround FAT32 4GB limitation (feature not exist in OpenShot yet but who knows what users will think about when reading the options names).

About the dark theme and name length the `Streamable (MP4):` - is shortest can think of. Not sure how it will be translated into other languages.